### PR TITLE
Improving the new read-the-docs (rtd) documentation

### PR DESCRIPTION
**Pull request by markdoerr**
This pull request will be used to suggest improvements of the new documentation.

... I will add more soon :)

### CODE REVIEW

**Comment by markdoerr** [comment]
Hi @AMontagu and @legau, 
I made some suggestions for an improved understanding in the "getting-started documentation" in the [2c60225](https://github.com/socotecio/django-socio-grpc/pull/206/commits/2c6022548b91b11ee45989ea4f6a9150c5d206f5) commit.

**Comment by markdoerr** [comment]
@AMontagu and @legau,
I added now some suggestions on how to improve the grpc-action documentation. 
In some places one of you should add a few more explanations for newcomers to the DSG framework ( I marked them with :TODO: ).

**Comment by markdoerr** [comment]
Dear @AMontagu and @legau,

I added some more comments / ideas / suggestions (also :TODO: s).

For the filters / search, a client example should also be added (to illustrate, how this works) - I added one in the filters.rst, but it could be improved.

Overall the documentation improved a lot and many open questions were answered :) - shall we also add a walk-through to the bibliography example to the read-the-docs page ? This could make things even clearer - but is also quite some work....

File: docs/features/proto-serializers.rst [hunk 1]
```diff
@@ -20,13 +30,83 @@ Mapping between Django REST Framework and Django Socio gRPC
      - ``django_socio_grpc.proto_serializers.ProtoSerializer``
    * - ``rest_framework.serializers.ListSerializer``
      - ``django_socio_grpc.proto_serializers.ListProtoSerializer``
-   * - ``rest_framework.serializers.ModelSerializer``
+   * - ``rest_framework.serializers.ModelProtoSerializer``
```
Comment (AMontagu) [inline docs/features/proto-serializers.rst:33]: this is not True.
rest_framework doesn't have protoSerilializer

File: docs/getting-started.rst [hunk 2]
```diff
@@ -45,7 +45,7 @@ Add now the following lines to the ``INSTALLED_APPS`` section of your ``tutorial
     'django_socio_grpc',
   ]
 
-See `Django tutorial <https://docs.djangoproject.com/en/4.2/intro/tutorial01/>`_ for more information 
+See `Django tutorial <https://docs.djangoproject.com/en/4.2/intro/dsg_tutorial01/>`_ for more information
```
Comment (AMontagu) [inline docs/getting-started.rst:48]: this is false but should be a find and replace

File: docs/getting-started.rst [hunk 3]
```diff
@@ -79,16 +79,16 @@ Finally migrate the database:
 
   python manage.py migrate
 
-See `Django tutorial <https://docs.djangoproject.com/en/4.2/intro/tutorial01/>`_ for more information 
+See `Django tutorial <https://docs.djangoproject.com/en/4.2/intro/dsg_tutorial01/>`_ for more information
```
Comment (AMontagu) [inline docs/getting-started.rst:82]: here too

File: docs/how-to/define-proto-and-service-in-a-shared-library.rst [hunk 4]
```diff
@@ -39,4 +43,64 @@ In this specific case, my_external_library is an external library.
       app_registry.register(ExternalService)
 
 By defining here, "to_root_grpc=True", when you generate your protos, they will be added in a folder with the name and path defined in your 
-setting grpc_settings.ROOT_GRPC_FOLDER. (by default grpc_folder at the root of your application).
\ No newline at end of file
+setting grpc_settings.ROOT_GRPC_FOLDER. (by default grpc_folder at the root of your application).
+
+
+# :TODO: 
+Comment:
```
Comment (AMontagu) [inline docs/how-to/define-proto-and-service-in-a-shared-library.rst:50]: remove this comment and put it in a other PR to discuss on it the time we figure out what the recommendations

File: docs/how-to/grpc-web.rst [hunk 5]
```diff
@@ -1,6 +1,6 @@
 .. _how-to-web:
-Web
-===
+pRPC-Web
```
Comment (AMontagu) [inline docs/how-to/grpc-web.rst:2]: pRPC ?

**Review by AMontagu — COMMENTED** [review]
Will merge as there is a lot of imporvements and personnally change the small diff

**Pull request merged**